core/cap-0038.md

```diff
@@ -569,7 +571,8 @@ index 75f39eb4..0b21aaab 100644
 +    LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED = -4,    // not enough balance for one of
 +                                                // the assets
 +    LIQUIDITY_POOL_DEPOSIT_BAD_PRICE = -5,      // deposit price outside bounds
-+    LIQUIDITY_POOL_DEPOSIT_POOL_FULL = -6       // pool reserves are full
++    LIQUIDITY_POOL_DEPOSIT_POOL_FULL = -6,       // pool reserves are full
++    LIQUIDITY_POOL_DEPOSIT_DISABLED = -7        // deposit is disabled
```

Comment:
Why do we need a new error code for this? I think we should check this at validation time and just use `MALFORMED`. Same applies for withdraw.

Reply:
Using `MALFORMED` here can be confusing for the user. I guess this is an obscure case though and we'll have bigger things to worry about if we needed to enable one of the flags. I'm fine going either way.

Reply:
Maybe @MonsieurNicolas has thoughts on this?

Reply:
I agree with you that it could be confusing, and I also agree that we would have bigger things to worry about.

Reply:
Updated to return `MALFORMED` at validation for now. We can update this later if we want.

Reply:
It's better to use `opNOT_SUPPORTED` that actually means "not supported at this time"

Reply:
Added a fixup commit to return `opNOT_SUPPORTED`.